### ISSUE TITLE
[Suggestion]: Improved Instance Cloning Options and Clarity

### ISSUE THREAD

**Issue by hk21702** [RimSort/RimSort#604]
### Origin of idea

Currently, the dialogues and true behavior of the instance cloning functionality is not intuitive or clear.

This is the current dialog for cloning instances, not including the dialog for naming the new instance:
![image](https://github.com/user-attachments/assets/7670a3b6-6059-4c72-9399-f521e5613e8a)

In current behavior:
1. if the game folder exists, RimWorld and local mods since by default local mods exist in the main install, it is cloned into the instance folder. Same with the config folder.

2. Then, local mods is copied again into the instance folder separately (I'm not sure if this is necessary? From my knowledge, RimWorld only handles local mods from the mods directory in the root of the RimWorld install. And, if the local mods the default — within the root of the game directory — then the previous step would have copied them anyway)

3. Then the user is given the option to convert steamworkshop mods into steamCMD mods. If they say yes, then workshop mods are copied into the new local mods folder.

4. steamCMD folders are copied if they exist (Note that this only check for the dir, not an actual steamCMD install when determining if they should be copied)

4.5. The symlink is checked. If it does exist, it is recreated to the new target local folder for that instance. If it does not exist, then it is created. 

5. The new instance object is created, if necessary, the user is asked if they'd like to auto-update their run arguments so that the savedatafolder arg is corrected:
![image](https://github.com/user-attachments/assets/8137ead3-39df-4d31-89e1-ba5f7c911fd3)

### Suggestion/proposed solution?

 - [ ] Clarify in the dialogue that instance cloning will clone the game and mod data as well as just the instance data, and that this may take a while
 - [ ] Prevent redundancy when cloning the game folder. Don't also clone the local mods. Alternatively, remove the local mods clone step (Also we may want to consider deprecating the ability to select the local mods folder all together, and just base it off where the game folder is set)
 - [ ] When asking to convert workshop mods to steamCMD mods, clarify that the instance will also ignore the workshop folder such that there are no duplicates. Add that to change this, re-apply the workshop path in settings. 
 - [ ] Provide a preview of what the auto-generated run args will be in the details of the dialogue asking if the user would like to utilize them.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Duplicate Issue Check

- [X] I've checked for duplicate issues

### Wiki/FAQ Check

- [X] I've checked the Wiki for a solution
